src/lib/spinel/spinel.h

```diff
@@ -4763,6 +4763,16 @@ enum
      */
     SPINEL_PROP_INFRA_IF_RECV_ICMP6 = SPINEL_PROP_INFRA_IF__BEGIN + 2,
 
+    /// ICMP6 message sent by NCP and needs to be sent on the infrastructure interface.
+    /** Format: `L6d`
+     * Type: Unsolicited notifications only
+     *
+     * `L`: The infrastructure interface index.
+     * `6`: The IP6 source address of the message to send.
```

Comment:
The function `InfraIfSendIcmp6Nd()` sets the IPv6 address as the destination address. Should this address be the destination address?

Reply:
Good catch! Updated.